### PR TITLE
fix: use mocked time to solve happo diffs

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
ZH: https://github.com/bcgov/cas-cif/issues/929

### CODE REVIEW

**Review by BCerki — APPROVED** [review]

**Comment by Sepehr-Sobhani** [comment]
@BCerki after looking at happo diffs, I found an issue with the mocked time I've implemented; I've added mocked time as today but `Complete` badge is showing yesterday. I think I need to change the mocked time to a full timestamptz.

File: app/cypress/support/commands.js [hunk 1]
```diff
@@ -52,11 +52,22 @@ Cypress.Commands.add("mockLogin", (roleName) => {
 });
 
 Cypress.Commands.add("sqlFixture", (fixtureName) => {
-  return cy.fixture(`${fixtureName}.sql`).then((fixture) =>
-    cy.exec(`psql -v "ON_ERROR_STOP=1" -d cif<< 'EOF'
+  cy.getCookies().then((cookies) => {
+    let mockedTimeCookie = cookies.filter(
```
Comment (pbastia) [inline app/cypress/support/commands.js:56]: this can be a const too

**Review by pbastia — CHANGES_REQUESTED** [review]
super nitpicky comment ;) Looks great, thank you!

File: app/cypress/support/commands.js (hunk 1 above)
Comment (Sepehr-Sobhani) [reply app/cypress/support/commands.js]: Done

**Review by pbastia — DISMISSED** [review]

**Review by pbastia — APPROVED** [review]
💯

**Pull request merged**